deploy/table_address.sql

```diff
@@ -7,8 +7,7 @@ create table ggircs.address (
 
     id                                                  integer primary key,
     report_id                                           integer references ggircs.report(id),
-    single_facility_id                                  integer references ggircs.single_facility(id),
```

Comment:
What else is this PR accomplishing besides unsplitting? Removing columns, especially foreign key constraints is an API-breaking change.

Reply:
This also has all of the Carbon Tax stuff Dylan worked on